cluster/addons/volumesnapshots/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -2,12 +2,9 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  labels:
-    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
Don't remove it. Addons with label `addonmanager.kubernetes.io/mode=Reconcile` will be periodically
reconciled by addonmanager.